lib/ktsllex/schemas.ex

```diff
@@ -97,16 +103,27 @@ defmodule Ktsllex.Schemas do
   end
 
   defp update_connect_name(:error, _schema_name, _type), do: :error
-  defp update_connect_name(schema, _schema_name, "-value"), do: schema
+  defp update_connect_name(schema, _schema_name, "value"), do: schema
 
-  defp update_connect_name(schema, schema_name, "-key") do
+  defp update_connect_name(schema, schema_name, "key") do
     schema
     |> Map.put("connect.name", schema_name <> ".Key")
   end
 
   defp read_schema(base_schema_file, type) do
-    (base_schema_file <> type <> ".json")
-    |> FileJson.read!()
+    json_file =
+      (base_schema_file <> ".json")
+      |> FileJson.read!()
+
+    case json_file do
+      %{"key_avro_schema" => key_schema, "value_avro_schema" => value_schema} ->
+        case type do
+          "key" -> key_schema
+          "value" -> value_schema
+        end
+      _ ->
+        {:error, :enoent}
```

Comment:
Maybe `enoent` should be more related to invalid schema defined? As it couldnt find both of those keys.